components/molecules/FavoriteRepoCard/favorite-repo-card.tsx

```diff
@@ -0,0 +1,42 @@
+import Avatar from "components/atoms/Avatar/avatar";
+import { truncateString } from "lib/utils/truncate-string";
+import { StaticImageData } from "next/image";
+import Link from "next/link";
+import React from "react";
+
+export interface FavoriteRepoCardProps {
+  avatarURL?: string | StaticImageData;
+  name?: string;
+  owner?: string;
+  topic?: string;
+  userPage?: string;
+}
+
+const FavoriteRepoCard = ({ avatarURL, name, owner, topic, userPage }: FavoriteRepoCardProps): JSX.Element => {
+  return (
+    <div className="p-3 bg-white border-2 rounded-xl">
+      <div className="flex items-center gap-2.5 h-10">
+        {/* Avatar */}
+        <a href={`https://www.github.com/${owner}/${name}`} target="_blank" rel="noreferrer" className="h-10">
+          <Avatar className="shrink-0 min-w-10 min-h-10" size={40} avatarURL={avatarURL} isCircle={false} />
+        </a>
+
+        {/* Text */}
+        <div className="flex flex-col justify-center">
+          <div className="  text-sm text-light-slate-11 truncate max-w-[85px] md:max-w-[110px]">
+            <Link href={`/${userPage ? `pages/${userPage}/` : ""}${topic}/repositories/filter/${owner}/${name}`}>
+              <a>{owner ? `${owner}` : "handle1234"}</a>
```

Comment:
not sure if wrapping another anchor inside `Link` is needed

Reply:
The adding of <a> to Link in Table Repository Names was included in the PR https://github.com/open-sauced/insights/pull/727/files and I copied that part as a whole

Reply:
You are right on the link usage, the rework for https://github.com/vercel/next.js/pull/36436 should be tackled in another issues as we are on `12.2.4`